libscript/tests/test_interpreter.cxx

```diff
@@ -53,7 +53,7 @@ TEST_CASE("Scripting-Refacto-Engine")
         "	# anyEntity.Position.pos_z = anyEntity.Position.pos_z + anyEntity.Velocity.vel_z# * deltaTime\n";
     component::Index cind;
     systems::Index sind;
-    script::Engine engine(sind, cind);
+    script::Engine engine(sind, cind, pivot::ecs::script::interpreter::builtins::BuiltinContext());
```

Comment:
Does this mean that all builtins will always use/receive the same context ?
Even if most of them don't use it ?

Reply:
Yes but they don't need to use it as you say.